ui/app/helpers/options-for-backend.js

```diff
@@ -8,7 +8,7 @@ const DEFAULT_DISPLAY = {
   create: 'Create secret',
   navigateTree: true,
   editComponent: 'secret-edit',
-  listItemPartial: 'partials/secret-list/item',
+  listItemPartial: 'secret-list/item',
```

Comment:
the naming for this key is no longer accurate, maybe we rename to `listItemComponent`?